proxy/ui.coffee

```diff
@@ -98,3 +107,14 @@ class WebExtUI extends UI
     super connected
     if connected then @stats[0] += 1
     @postActive()
+    if @active
+      chrome.browserAction.setIcon
+        path:
+          32: "icons/status-running.png"
```

Comment:
When `setActive` is called with `!connected` after already having been in the `connected` state, the icon is left as running.

It needs to return to `"on"`.

Reply:
Fixed.